pages/linux/dos2unix.md

```diff
@@ -0,0 +1,23 @@
+# dos2unix
+
+> DOS/Mac to UNIX and vice versa text file format converter.
+
+- Convert DOS file format to UNIX format:
+
+`dos2unix {{filename}}`
+
+- Convert DOS file format to UNIX format:
+
+`unix2dos {{filename}}`
```

Comment:
Let us not put 2 different commands in the same page. You can add a subheading at the top mentioning "Also see `unix2dos`"

Reply:
added